openeogeotrellis/integrations/kubernetes.py

```diff
@@ -76,16 +79,25 @@ def k8s_state_to_openeo_job_status(state: str) -> str:
     # TODO: is there a kubernetes state for canceled apps?
     return job_status
 
+
 def k8s_render_manifest_template(template, **kwargs) -> dict:
     import yaml
     """ Load and render a provided kubernetes manifest jinja template with the passed kwargs """
+    # TODO: move away from pkg_resources https://github.com/Open-EO/openeo-geopyspark-driver/issues/954
     jinja_path = pkg_resources.resource_filename(
         "openeogeotrellis.deploy", template
     )
     jinja_dir = os.path.dirname(jinja_path)
-    jinja_template = Environment(
+    jinja_env = Environment(
         loader=FileSystemLoader(jinja_dir)
-    ).from_string(open(jinja_path).read())
+    )
+
+    def base64encode(input_str: str) -> str:
+        return base64.b64encode(input_str.encode()).decode()
```

Comment:
```suggestion
        return base64.b64encode(input_str.encode("utf8")).decode("utf8")
```

just to be sure

Reply:
I was under the impression that utf-8 was the default but I don´t mind explicit so will change

Reply:
The default encoding depends on env vars and that kind of black magic. :magic_wand: 
So to avoid endless "works on my computer" debug sessions, I try to always be explicit about it

Reply:
Aha TIL, good to know I can imagine indeed it being a "time sink" debug session to troubleshoot something like that